draft-ietf-rats-architecture.md

```diff
@@ -1492,20 +1491,29 @@ in terms of `time(RX_v)-time(RG_v)`, then the Relying Party can check
 
 ## Example 3: Handle-based Passport Model Example
 
-The following example illustrates a hypothetical Passport Model
-solution that uses handles instead of nonces or timestamps.
-In this example, Evidence generation based on received handles always uses the current (most recent) handle.
-As handles are distributed over the network, all involved entities receive a fresh handle at roughly the same time.
-Due to distribution over the network, there is some jitter with respect to the time the Handle is received, time(HR), for each involved entity.
-To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR'_v) + offset - time(HR_v)`. The offset is typically based on a network's round trip time.
-Analogously, the generation of valid Evidence is only possible, if the age of the handle used is lower than the validity-duration: `time(HR_v) - time(EG_a) < validity-duration`.
+The example in {{fig-handle-passport}} illustrates a Handle-based interaction
+using the passport topology model.
```

Comment:
```suggestion
The example in {{fig-handle-passport}} illustrates a hypothetical Passport Model
solution that uses handles instead of nonces or timestamps.
```
Keep wording consistent with intros to the previous sections.